### PR TITLE
Release Google.Cloud.Tools.ReleaseProgressReporter version 0.2.0

### PR DESCRIPTION
**Pull request by jskeet**
(Again, this will need manual releasing.)

### CODE REVIEW

**Comment by jskeet** [comment]
I won't merge this until I've seen the DocUploader release complete successfully.

**Review by amanda-tarafa — APPROVED** [review]

**Comment by jskeet** [comment]
(Docuploader release failed, but that's fixed in #10968; will merge this and see if it works, while creating a docuploader 0.2.1 release.)

**Pull request merged**
